modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,118 @@
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+Use the following procedure to automatically discover local devices, and to automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name you want to open. The "Node Details" page is displayed.
+
+.. From the *Actions* drop-down list, select *Discover nodes* to allow automatic discovery of the selected  devices on the node.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list now updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node.
+====
+
+. To automatically provision local volumes for the discovered devices:
+
+.. From the CLI, create an object YAML file to define the local volume set, such as `local-volume-set.yaml`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeSet
+metadata:
+  name: example-autodetect
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - worker-0
+              - worker-1
+  storageClassName: example-storageclass <1>
+  volumeMode: filesystem
+  fstype: ext4
+  maxDeviceCount: 10
+  deviceInclusionSpec:
+    deviceTypes:
+      - RawDisk
+      - Disk
+      - Partition
+    deviceMechanicalProperty:
+    deviceMechanicalProperty:
+      - NonRotational
+    minSize: 10G
+    maxSize: 100G
+    models:
+      - SAMSUNG
+      - Crucial_CT525MX3
+    vendors:
+      - ATA
+      - ST2000LM
+status:
+  conditions:
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: 'DiskMaker: Available, LocalProvisioner: Available'
+    status: "True"
+    type: DaemonSetsAvailable
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: Operator reconciled successfully.
+    status: "True"
+    type: Available
+  observedGeneration: 1
+  totalProvisionedDeviceCount: 0
+----
++
+<1> Determines the dynamic provisioning directory.
+
+.. Create the local volume set object by entering the following command:
```

Comment:
similar to `LocalVolume` object - it should be documented that, deleting the LocalVolumeSet will not remove PVs and symlinks and should be manually removed - cc https://bugzilla.redhat.com/show_bug.cgi?id=1887574